src/Components/MovieForm/MovieForm.jsx

```diff
@@ -14,87 +14,158 @@ const defaultMovieValues = {
-              formGenres.map((genre) => (
-                <option key={genre.value} value={genre.value}>
-                  {genre.label}
-                </option>
-              ))}
-          </select>
-        </div>
-        <div className={styles.formGroup}>
-          <LabeledInput label="Runtime" onChange={handleChange} value={movie.runtime} id="runtime" />
-        </div>
-      </div>
+      {({ values, errors, touched, handleChange, handleBlur, handleSubmit, resetForm }) => (
+        <form className={styles.container} onSubmit={handleSubmit}>
+          <div className={styles.formRow}>
+            <div className={styles.formGroup}>
+              <LabeledInput
+                label="Title"
+                onChange={handleChange}
+                value={values.title}
+                id="title"
+                handleBlur={handleBlur}
+              />
+              <span className={styles.error}>{errors.title && touched.title && errors.title}</span>
+            </div>
+            <div className={styles.formGroup}>
+              <LabeledInput
+                label="Release Date"
+                onChange={handleChange}
+                value={values.releaseYear}
+                id="releaseYear"
+                handleBlur={handleBlur}
+              />
+              <span className={styles.error}>{errors.releaseYear && touched.releaseYear && errors.releaseYear}</span>
+            </div>
+          </div>
+          <div className={styles.formRow}>
+            <div className={styles.formGroup}>
+              <LabeledInput
+                label="Image Url"
+                onChange={handleChange}
+                value={values.imageUrl}
+                id="imageUrl"
+                handleBlur={handleBlur}
+              />
+              <span className={styles.error}>{errors.imageUrl && touched.imageUrl && errors.imageUrl}</span>
+            </div>
+            <div className={styles.formGroup}>
+              <LabeledInput
+                label="Rating"
+                onChange={handleChange}
+                value={values.rating}
+                id="rating"
+                handleBlur={handleBlur}
+                type="number"
+              />
+              <span className={styles.error}>{errors.rating && touched.rating && errors.rating}</span>
+            </div>
+          </div>
+          <div className={styles.formRow}>
+            <div className={styles.formGroup}>
+              <label className={styles.label} htmlFor="genre">
+                Genre
+              </label>
+              <select className={styles.select} id="genre" name="genre" value={values.genre} onChange={handleChange}>
+                <option hidden />
+                {!!formGenres?.length &&
+                  formGenres.map((genre) => (
+                    <option key={genre.value} value={genre.value}>
+                      {genre.label}
+                    </option>
+                  ))}
```

Comment:
I missed it in one of the previous pr-s, but this can be simplified:

```suggestion
                {formGenres?.map((genre) => (
                  <option key={genre.value} value={genre.value}>
                    {genre.label}
                  </option>
                ))}
```